post-office-stub/src/README.md

```diff
@@ -0,0 +1,10 @@
+# Gov Notify stub
+# Post Office Stub
+### Stub for the Post Office Location & Data Services (‘Branch Finder’) API service which mocks the Post Office POST (https://locations.pol-platform.co.uk/v1/locations/search) endpoint and responds with success or error responses.
+### Returns the following HTTP response based on the last 3 characters of the postcode request payload
+* No postcode suffix - Successful response
+* 400 - 400 response
+* 403 - 403 response
```

Comment:
I can't find the docs for the service that we are mocking (do they exist?) but do they not have a 404?

Reply:
Docs exist in the form of a PDF, not sure if they are available online, they don't mention any error codes

Reply:
Asked Sanka on the ticket, I would have thought it'd be the most common error?